Source/ChocolateyGui/Services/ChocolateyRemotePackageService.cs

```diff
@@ -242,7 +282,9 @@ public ValueTask<bool> RequiresElevation()
         public void Dispose()
         {
             _logStream?.Dispose();
-            _wampChannel?.Close("Exiting", new GoodbyeDetails { Message = "Exiting" });
+            _wampChannel?.Close(
+                Resources.ChocolateyRemotePackageService_Exiting,

```

Comment:
Actually, these need to stay hardcoded magic strings. I'll move them to a const later.